workflows/pipe-common/scripts/autoscale_sge.py

```diff
@@ -1002,9 +1048,11 @@ def scale_up(self, resource):
         Logger.info('Start grid engine SCALING UP.')
         return self.scale_up_handler.scale_up(resource)
 
-    def _scale_down(self, running_jobs, additional_hosts):
+    def _scale_down(self, running_jobs, additional_hosts, scaling_period_start=None):
```

Comment:
It's better to keep it the existing way. This single call performs better without `scaling_period_start` being specified because it only applies to deadlock scenarios.